Bottom: aeb9305e3535eefa172def2f985caca4ddfb058d
Top:    a1c9757586d5ae2af45e801d301014aaead72118
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-30 10:25:33 +0000

Refresh of change-found-objects-colour-fr

---

diff --git a/src/main.c b/src/main.c
index e41fc55..d80612d 100644
--- a/src/main.c
+++ b/src/main.c
@@ -770,11 +770,11 @@ Color to indicate physical connections. Default: @samp{#00ff00}
 /* %start-doc options "3 Colors"
 @ftable @code
 @item --found-color <string>
-Color to indicate logical connections. Default: @samp{#00ff00}
+Color to indicate logical connections. Default: @samp{#ff00ff}
 @end ftable
 %end-doc
 */
-  COLOR (FoundColor, "#00ff00", "found-color",
+  COLOR (FoundColor, "#ff00ff", "found-color",
 	 "color to indicate logically connected objects"),
 
 /* %start-doc options "3 Colors"
